### DISCUSSION TITLE
Προθεσμία για ακήσεις κειμένου

### DISCUSSION THREAD

**Discussion by AlexandrosV20** [courses-ionio/hci#1637] [Announcements]
Υπάρχει ένα πρόβλημα με τις ασκήσεις κειμένου του chapter 2 καθώς μου το βγάζει κλειδωμένο. Έπρεπε όντως να είναι κλειδωμένο ; Γιατι μου φαίνεται περίεργο γιατί θυμάμαι που λέγατε ότι συζητιούνται 5 ερωτήσεις στο μαθήμα και αυτη την βδομάδα που μας πέρασε είπατε οτι δεν θα γίνουν τα μαθήματα. @epidrome

**Comment by epidrome** [comment]
ενημερώθηκαν οι προθεσμίες ώστε το 2ο να είναι διαθέσιμο μέχρι 21 Δεκ

οι προθεσμίες δεν φαίνονται στο τερματικό σας;

**Reply by AlexandrosV20** [reply]
Ναι τωρα μολις το ειδα και στο δικο μου προφιλ οι προθεσμιες αλλαξαν μεχρι 21 Δεκεμβριου.
